ui/src/components/MainContainer.js

```diff
@@ -265,13 +266,19 @@ const MainContainer = () => {
           permissions={['t3ch', 'admin', 'stewardship']}
           component={FormManager}
         />
-        <PrivateRoute 
+        <PrivateRoute
+          exact
+          path="/admin/announcements"
+          permissions={['t3ch', 'admin', 'stewardship']}
```

Comment:
iirc we wanted vp to have readonly access to this page. should we create a new user group, or is that not in scope of this pr

Reply:
yea, i have a new issue for that PR. i think it gets too big to review if I combine everything here. wanna make your life easier too hehe